core/src/main/scala/com/evolutiongaming/catshelper/CatsHelper.scala

```diff
@@ -68,16 +68,13 @@ object CatsHelper {
 
   implicit class OpsCatsHelper[F[_], A](val self: F[A]) extends AnyVal {
 
-    def startEnsure(implicit F: Concurrent[F]): F[Fiber[F, A]] = F.startEnsure(self)
+    def startEnsure(implicit F: Concurrent[F]): F[Fiber[F, Throwable, A]] = F.startEnsure(self)
 
 
     def toTry(implicit F: ToTry[F]): Try[A] = F.apply(self)
 
 
     def toFuture(implicit F: ToFuture[F]): Future[A] = F.apply(self)
-
-
-    def toResource(implicit F: Applicative[F]): Resource[F, A] = Resource.liftF(self)
```

Comment:
you can deprecate, but not delete :) please

Reply:
I removed it because there's an identical method in `cats.effect.kernel.syntax.EffectResourceOps` which clashes with our own, so there's no need for that when using CE3